templates/taxi/confirm_delete_car.html

```diff
@@ -0,0 +1,11 @@
+{% extends "base.html" %}
+{% load crispy_forms_filters %}
+{% block content %}
+  <p>Do you want to delete?{{ car }}</p>
+  <p><i>Everything what bundle with this car will be also deleted!</i></p>
+  <form action="" method="post">
+  {% csrf_token %}
+    <input type="submit" value="Confirm" class="btn btn-danger">
+    <a href="{% url 'taxi:car-list' %}" class="btn btn-secondary">Cansel</a>
```

Comment:
There is a typo in the cancel button text. It should be 'Cancel' instead of 'Cansel'.